app/views/shared/xml/_activity.xml.haml

```diff
@@ -5,7 +5,7 @@
   %title
     %narrative= activity.title
   %description{"type" => "1"}
-    %narrative= activity.covid19_related.zero? ? activity.description : "#{activity.description} COVID-19"
```

Comment:
Are all of the other `nil` attributes for non-ODA activities handled?

Reply:
Anecdotally yes, since this is the only one that caused errors when viewing a NODA activity as XML.